src/i18n.js

```diff
@@ -41,7 +41,8 @@ export function startClient() {
 	});
 }
 
-const DOCUMENT_REGEX = /^([^.?#@]+)?([?#](.+)?)?$/;
+// init only for routes (urls with no extensions / .js, .css, ... /) and a service worker
+const DOCUMENT_REGEX = /(\/([\w-]+)?|^\/service-worker-index\.html)($|[?#]([\w-]+)?)/;
```

Comment:
I'm not sure about this, but is it possible for the service-worker URL to have something after the `.html`?

```suggestion
const DOCUMENT_REGEX = /(^([^.?#@]+)?([?#](.+)?)?|service-worker.*?\.html)$/;
```

Reply:
I'm not sure either... Anyway let's keep your solution...)